### ISSUE TITLE
Update published version on crates.io

### ISSUE THREAD

**Issue by tribals** [kriomant/ogrep-rs#4]
Currently available version (which is 0.2.1) is broken:

```console
$ cargo install ogrep
    Updating crates.io index
  Installing ogrep v0.2.1
   Compiling libc v0.2.62
   Compiling memchr v2.2.1
   Compiling bitflags v1.1.0
   Compiling ucd-util v0.1.5
   Compiling unicode-width v0.1.6
   Compiling regex v0.2.11
   Compiling lazy_static v1.4.0
   Compiling utf8-ranges v1.0.4
   Compiling ansi_term v0.11.0
   Compiling strsim v0.8.0
   Compiling either v1.5.3
   Compiling vec_map v0.8.1
   Compiling ansi_term v0.10.2
   Compiling regex-syntax v0.5.6
   Compiling textwrap v0.11.0
   Compiling thread_local v0.3.6
   Compiling itertools v0.7.11
   Compiling aho-corasick v0.6.10
   Compiling atty v0.2.13
   Compiling clap v2.33.0
   Compiling ogrep v0.2.1
error[E0283]: type annotations required: cannot resolve `std::borrow::Cow<'_, str>: std::convert::From<&_>`
   --> /home/tosh/.cargo/registry/src/github.com-1ecc6299db9ec823/ogrep-0.2.1/src/main.rs:541:17
    |
541 |                 Cow::from(options.pattern.as_ref())
    |                 ^^^^^^^^^
    |
    = note: required by `std::convert::From::from`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0283`.
error: failed to compile `ogrep v0.2.1`, intermediate artifacts can be found at `/tmp/cargo-installduKz53`

Caused by:
  Could not compile `ogrep`.

To learn more, run the command again with --verbose.
$ cargo --version
cargo 1.37.0
$ rustc --version
rustc 1.37.0
```

**Comment by kriomant** [comment]
Published 0.4.0

**Comment by tribals** [comment]
Thanks!
